CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Exclude expired certificates in secrets for `CertificateSecretWillExpireInLessThanTwoWeeks`.
```

Comment:
```suggestion
- Exclude expired certificates in secrets for `CertificateSecretWillExpireInLessThanTwoWeeks`, to reduce operational toil of non-actionable, non GS-managed certificates.
```

Reply:
Thats not entirely true. This has nothing to do with non GS-managed certs